pkg/services/ngalert/sender/router.go

```diff
@@ -116,13 +115,13 @@ func (d *AlertsRouter) SyncAndApplyConfigFromDatabase() error {
 
 		// We have no running sender and no Alertmanager(s) configured, no-op.
 		if !ok && len(cfg.Alertmanagers) == 0 {
-			d.logger.Debug("no external alertmanagers configured", "org", cfg.OrgID)
```

Comment:
I think it should be a debug because this is the normal path and it can be noisy